api/models/storeDao.js

```diff
@@ -0,0 +1,28 @@
+const appDataSource = require('./dataSource')
+
+const getAllstores = async () => {
+  try{
+    const allStoreData = await appDataSource.query(
+      `
+      SELECT
+        s.name,
+        s.thumbnail_image,
+        c.type,
+        s.social_map_id
+      FROM 
+        stores as s
+      INNER JOIN
+        categories as c
+      ON 
+        s.category_id = c.id
+      `
+    )
+    return allStoreData
```

Comment:
```suggestion
    return await appDataSource.query(
      `
      SELECT
        s.name,
        s.thumbnail_image,
        c.type,
        s.social_map_id
      FROM 
        stores as s
      JOIN
        categories as c
      ON 
        s.category_id = c.id
      `
    )
```
불필요한 변수 선언 없이 바로 return하면 될 것 같아요! 또한 INNER는 생략될 수 있으므로 안써주셔도 됩니다.